test/recording.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  setupRecording,
+  Recording,
+  SetupRecordingInput,
+} from '@jupiterone/integration-sdk-testing';
+
+export { Recording };
+
+export function setupMimecastRecording(
+  input: Omit<SetupRecordingInput, 'mutateEntry'>,
+): Recording {
+  return setupRecording({
+    ...input,
+    options: {
+      matchRequestsBy: {
+        url: {
+          hostname: false,
+        },
+      },
+      recordFailedRequests: false,
+    },
+    redactedRequestHeaders: ['Authorization', 'x-mc-app-id'],
```

Comment:
redact all headers (save for timestamp) that are used for authorization, or in the construction of the auth token.